docs/intro.rst

```diff
@@ -0,0 +1,103 @@
+Camus
+================
+
+Camus is a pipeline from Kafka to HDFS.
```

Comment:
How about - "Camus is a simple MapReduce job to load data from Kafka into HDFS. It is capable of incrementally copying data form Kafka into HDFS such that every run of the MapReduce job picks up from where the previous run left off. At LinkedIn, Camus is used to load billions of messages per day from Kafka into HDFS. Confluent's Camus distribution will work with CDH x"

We have to be careful not to imply that we "certify" against CDH. So we can say something like it will work with CDH x.